pkg/utils/utils.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"crypto/sha512"
+	"fmt"
+)
+
+const ChainCNIPrefix = ("CNI-")
+const MaxChainLength = 29 - len(ChainCNIPrefix)
```

Comment:
We don't, good point! I'll also rename it as I don't like the name anymore today.